frontend/yarn.lock

```diff
@@ -9542,6 +9542,11 @@ send@0.17.1:
     range-parser "~1.2.1"
     statuses "~1.5.0"
 
+serialize-javascript@^1.7.0:
```

Comment:
I found this version also needs to exist for now. Maybe we need to deal with dependencies between versions.